src/xattrs.c

```diff
@@ -37,6 +37,14 @@ enum xattrs_action_type_t_ {
 
 typedef enum xattrs_action_type_t_ xattrs_action_type_t;
 
+/* If SWUPD_WITH_XATTRS is not defined, create functions to override
+ * glibc's interface to the system calls
+ */
+#ifndef SWUPD_WITH_XATTRS
+#define lgetxattr(p,n,b,l) (-1)
+#define llistxattr(p,b,l) (0)
```

Comment:
If I am going to return -1, I should also set errno. So I am saying there are zero attributes, rather than saying there is an error trying to get them (perhaps ENOTSUP). As you can tell I am in two minds about which is the correct approach.